README.md

```diff
@@ -10,14 +10,15 @@ The Node.js End User Feedback team works with Node.js end users to provide a bi-
 * Support [CommComm](https://github.com/nodejs/community-committee) and [TSC](https://github.com/nodejs/community-committee) end user feedback and outreach needs.
 
 ### Governance and Contributing
-The Node.js End User Feedback team has adopted the core governance and contributing policies of Node.js.
+The Node.js End User Feedback team has adopted the core governance and contribution policies of the Node.js project.
+ 
+The Node.js End User Feedback team is chartered under the [Node.js Community Committee (CommComm)](https://github.com/nodejs/community-committee) and follows [CommComm Governance policy](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md).
```

Comment:
Chartering is something that only Committees and Working groups have- so we'll need different wording here.

Reply:
Please review the existing language. This "chartering" terminology is only being moved up to make more space for expanded text around contributing.

Reply:
@williamkapke @bnb Please submit a separate issue if you desire to address the "chartered" nature of this initiative or otherwise. The text exists in the [current readme](https://github.com/nodejs/user-feedback/blob/9242ecea677f891c9e5426f2ad8f467d0f2faacd/README.md) and has only been moved in this update.

Reply:
@dshaw once this lands, can you submit a PR to remove it? So:

`is chartered under` > `is an initiative of`

Reply:
@dshaw can you PR an update once this PR lands? So:

`is chartered under` > `is an initiative of`

I don't see an issue making this change in this PR, but if you feel the need to move it into another PR then feel free to 👍